R/wrangle_and_check.R

```diff
@@ -0,0 +1,31 @@
+#' Wrangles and checks sector exposures
+#'
+#' Wrangles sector exposures by removing invalid rows and rows from assets other
+#' than `asset_types`. Also checks for duplicates and that more than 0 rows
+#' remain.
+#'
+#' @param sector_exposures A tibble holding sector_exposures as calculated by
+#'   PACTA.
+#' @param asset_type Type of analyzed asset. Can be bonds or equity.
+#'
+#' @return `sector_exposures` holding only valid rows for `asset_type`
+#' @export
+wrangle_and_check_sector_exposures_eq_cb <- function(sector_exposures, asset_type) {
+
+  if (!asset_type %in% c("Bonds", "Equity")) {
```

Comment:
are you not doing this for loans, because there is no such flag there? (I think it's correct, just checking if this is assumed?)

Reply:
Excactly, 

- there both is no flag on invalid_input 
- only 1 asset type is included so no need for filtering.

Thus one has to trust that dealing with valid_input (if necessary) was done correctly whereever it is done. However since I will add a check for duplicates we should notice if valid_input filtering was not done.